src/jmclient/wallet_utils.py

```diff
@@ -721,10 +721,17 @@ def cli_do_support_fidelity_bonds() -> bool:
         jmprint("Not supporting fidelity bonds", "info")
         return False
 
-def wallet_generate_recover_bip39(method, walletspath, default_wallet_name,
-        display_seed_callback, enter_seed_callback, enter_wallet_password_callback,
-        enter_wallet_file_name_callback, enter_if_use_seed_extension,
-        enter_seed_extension_callback, enter_do_support_fidelity_bonds, mixdepth=DEFAULT_MIXDEPTH):
+def wallet_generate_recover_bip39(method: str,
+                                  walletspath: str,
+                                  default_wallet_name: str,
+                                  display_seed_callback: Callable[[str, str], None],
+                                  enter_seed_callback: Optional[Callable[[], Tuple[Optional[str], Optional[str]]]],
+                                  enter_wallet_password_callback: Callable[[], str],
+                                  enter_wallet_file_name_callback: Callable[[], str],
+                                  enter_if_use_seed_extension: Optional[Callable[[], bool]],
+                                  enter_seed_extension_callback: Optional[Callable[[], Optional[str]]],
+                                  enter_do_support_fidelity_bonds: Callable[[], bool],
+                                  mixdepth: int = DEFAULT_MIXDEPTH) -> bool:
```

Comment:
Shouldn' t we use `black` to consistently format code?

Reply:
What is that? Have no experience with it.

Reply:
It is a Python code formatter. See https://black.readthedocs.io/en/stable/
It can be integrated to most IDE or via a [pre-commit hook](https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/) 

Real nice I have to say, you just don't have to bother about formatting any-longer, it does it all for you :)

Reply:
Probably something to consider, but I don't see it as a top priority.